src/components/cards/LargeHorizontalCard/LargeHorizontalCard.jsx

```diff
@@ -18,10 +31,32 @@ function LargeHorizontalCard({ video }) {
             alt="plant image"
           />
 
+          {location.pathname == "/historyVideos" ? (
```

Comment:
suggestion: use if else statements if there are nesting conditions.

Reply:
Its if else only with ternary operator , its easier to use in jsx